File: data/EXP01/python/2348/2348.output_1.py
# @lc app=leetcode id=2348 lang=python3
class Solution:
    def zeroFilledSubarray(self, nums: List[int]) -> int:
        count = 0
        subarray_zeros = 0
        for num in nums:
            if num == 0:
                subarray_zeros += 1
                count += subarray_zeros
            else:
                subarray_zeros = 0
        return count

File: data/EXP01/go/2348/2348.output_1.go
func zeroFilledSubarray(nums []int) int64 {
    count := int64(0)
    for i := 0; i < len(nums); i++ {
        if nums[i] == 0 {
            count += int64(subarrayCount(nums[i+1:]))
        }
    }
    return count
}

func subarrayCount(nums []int) int {
    count := 0
    for i := 0; i < len(nums); i++ {
        if nums[i] != 0 {
            break
        }
        count++
    }
    return count
}
